docs/docs/llms/llm-nlg.mdx

```diff
@@ -0,0 +1,263 @@
+rephrase the template given the current conversation. The rephrased prompt is
+then returned as the response instead of the predicted template response.
+
+## Customizing
+
+You can customize the LLM by modifying the following parameters in the
+`endpoints.yml` file.
+
+### Rephrasing all responses
+
+Instead of enabling rephrasing per response, you can enable it for all responses
+by setting the `rephrase_all` property to `true` in the `endpoints.yml` file:
+
+```yaml-rasa title="endpoints.yml"
+nlg:
+    type: rasa_plus.ml.LLMResponseRephraser
+    rephrase_all: true
+```
+
+Defaults to `false`. Setting this property to `true` will enable rephrasing for
+all responses, even if they don't specify `rephrase: true` in the response
+metadata. If you want to disable rephrasing for a specific response, you can set
+`rephrase: false` in the response metadata.
+
+### OpenAI Model
+
+You can specify the openai model to use for rephrasing by setting the
+`model_name` property in the `endpoints.yml` file:
+
+```yaml-rasa title="endpoints.yml"
+nlg:
+    type: rasa_plus.ml.LLMResponseRephraser
+    model_name: text-davinci-003
+```
+
+Defaults to `text-davinci-003`. The model name needs to be set to a generative
+model using the completions API of
+[OpenAI](https://platform.openai.com/docs/guides/gpt/completions-api).
+
+### Temperature
+
+The temperature allows you to control the diversity of the generated responses.
+You can specify the temperature to use for rephrasing by setting the
+`temperature` property in the `endpoints.yml` file:
+
+```yaml-rasa title="endpoints.yml"
+nlg:
+    type: rasa_plus.ml.LLMResponseRephraser
+    temperature: 0.7
+```
+
+Defaults to `0.7`. The temperature is a value between 0 and 1 that controls the
+diversity of the generated responses. Lower temperatures result in more
+predictable responses, while higher temperatures result in more surprising
+responses.
+
+### Prompt
+
+You can change the prompt used to rephrase the response by setting the `prompt`
+property in the `endpoints.yml` file:
+
+```yaml-rasa title="endpoints.yml"
+nlg:
+    type: rasa_plus.ml.LLMResponseRephraser
+    prompt: |
+        The following is a conversation with
+        an AI assistant. The assistant is helpful, creative, clever, and very friendly.
+        Rephrase the AI response staying close to the original message and retaining
+        its meaning. Use simple english.
+
+        Summary of the conversation:
+        {{history}}
+
+        {{current_input}}
+        AI Response: {{suggested_response}}
+
+        Rephrased Response:
+```
+
+The prompt is a [Jinja2](https://jinja.palletsprojects.com/en/3.0.x/) template
+that can be used to customize the prompt. The following variables are available
+in the prompt:
+
+- `history`: The conversation history as a summary of the prior conversation,
+  e.g.
+  ```
+  User greeted the assistant.
+  ```
+- `current_input`: The current user input, e.g.
+  ```
+  HUMAN: I want to open a bank account
```

Comment:
does `HUMAN` need to be specified or is it already contained in `{{ current_input }}`?